### DISCUSSION TITLE
installation a32nx

### DISCUSSION THREAD

**Discussion by cocothebe** [flybywiresim/aircraft#3501] [Help] [answered]
I tried to instal the folder. The installer put the folder on c:games\flightsim..\hmlpack..\community, but when i launch flighsimulator the display of a320 it doesn't work as the rest of buttons inside. If i want it works again i must delete the folder a32nx.  i tried also to put the a32nx folder manually but still doesn't work. How can i solve this problem?? thanks

**Comment by robinottawa** [comment, marked as answer]
There is a program that will install it for you, and replace it and update it when there is a different version available (you still have to open the installer app.). Have you seen it - The installer app?

**Reply by cocothebe** [reply]
I tried but it didn't work, the display are still blank. The a32nx doesn't work, I tried everything
